classrom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    @students.push(student)
```

Comment:
* Kindly add the `has many` association to this method that is When adding a student to a classroom it also sets the classroom for the student HINT 💭 you can wirte something like this ` student.classroom = self` so that any time we add a student to a classroom the classroom of the student is set as well, and vice versa [Here is a link to articles with examples of has-many-relationship](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#:~:text=class%20Owner%0A%20%20...%0A%0A%20%20def,self)%0A%20%20end%0A%0A%20%20...%0Aend)

Reply:
Resolved